src/amr/data/field/initializers/field_user_initializer.hpp

```diff
@@ -0,0 +1,39 @@
+#ifndef _PHARE_AMR_DATA_FIELD_INITIAZILIZERS_FIELD_USER_INITIALIZER_HPP_
+#define _PHARE_AMR_DATA_FIELD_INITIAZILIZERS_FIELD_USER_INITIALIZER_HPP_
+
+#include "core/utilities/span.hpp"
+#include "initializer/data_provider.hpp"
+
+#include <tuple>
+#include <memory>
+
+namespace PHARE 
+{
+namespace core{    
```

Comment:
small nitpick

we can minimize the indentation and reduce needs for new lines by the following

```
namespace PHARE::core {

...

} // namespace PHARE::core
```